formats/IO.json

```diff
@@ -1,7 +1,7 @@
 {
   "Description": "IO : BBND 1ZZ",
   "RedundantCharacters": " -",
-  "ValidationRegex": "^BBND1ZZ$",
+  "ValidationRegex": "^BBND ?1ZZ$",
```

Comment:
Aren't we removing the space? This should be the purpose of RedundantCharacters on line 3.

Reply:
I got it now, a replace needs to be run fist before regex. Thanks.